src/NzbDrone.Core/DecisionEngine/DownloadDecisionComparer.cs

```diff
@@ -0,0 +1,151 @@
+            {
+                var delayProfile = _delayProfileService.BestForTags(remoteEpisode.Series.Tags);
+                var downloadProtocol = remoteEpisode.Release.DownloadProtocol;
+                return downloadProtocol == delayProfile.PreferredProtocol;
+            });
+
+            return result;
+        }
+
+        private int CompareEpisodeCount(DownloadDecision x, DownloadDecision y)
+        {
+            return Compare(Compare(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode => remoteEpisode.ParsedEpisodeInfo.FullSeason),
+                           CompareReverse(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode => remoteEpisode.Episodes.Count));
+        }
+
+        private int CompareEpisodeNumber(DownloadDecision x, DownloadDecision y)
+        {
+            return CompareReverse(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode => remoteEpisode.Episodes.Select(e => e.EpisodeNumber).MinOrDefault());
+        }
+
+        private int ComparePeers(DownloadDecision x, DownloadDecision y)
+        {
+            // Different protocols should get caught when checking the preferred protocol,
+            // since we're dealing with the same series in our comparisions
+            if (x.RemoteEpisode.Release.DownloadProtocol != DownloadProtocol.Torrent ||
+                y.RemoteEpisode.Release.DownloadProtocol != DownloadProtocol.Torrent)
+            {
+                return 0;
+            }
+
+            return Compare(
+                Compare(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode =>
+                {
+                    var seeders = TorrentInfo.GetSeeders(remoteEpisode.Release);
+
+                    return seeders.HasValue ? Math.Round(Math.Log10(seeders.Value)) : 0;
+                }),
+                Compare(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode =>
+                {
+                    var peers = TorrentInfo.GetPeers(remoteEpisode.Release);
+
+                    return peers.HasValue ? Math.Round(Math.Log10(peers.Value)) : 0;
+                }));
+        }
+
+        private int CompareAge(DownloadDecision x, DownloadDecision y)
+        {
+            if (x.RemoteEpisode.Release.DownloadProtocol != DownloadProtocol.Usenet ||
+                y.RemoteEpisode.Release.DownloadProtocol != DownloadProtocol.Usenet)
+            {
+                return 0;
+            }
+
+            return Compare(x.RemoteEpisode, y.RemoteEpisode, remoteEpisode =>
+            {
+                var ageHours = remoteEpisode.Release.AgeHours;
+                var age = remoteEpisode.Release.Age;
+
+                if (ageHours < 1)
+                {
+                    return 1000;
+                }
+
+                if (ageHours <= 24)
+                {
+                    return 100;
+                }
+
+                if (age <= 7)
+                {
+                    return 10;
+                }
+
+                return 1 - age/365;
```

Comment:
This means stuff older than 7 days rarely gets compared by size. Personally I would throw them all one one pile.